contributing.md

```diff
@@ -33,4 +33,4 @@ inclusion.
 
 ## Links ##
 
-See also the [Contributing to Refinery](http://refinerycms.com/guides/contributing-to-refinery) guide.
+See also the [Contributing to Refinery](/guides/contributing-to-refinery) guide.
```

Comment:
this will 404 if you're viewing this on GitHub, which you probably are 😄

Reply:
Fixed

Reply:
I don't see the change? The link should be: `https://www.refinerycms.com/guides/contributing-to-refinery`

Reply:
Sorry it's the only one i've missed.